src/app/enums/default-validation-error.enum.ts

```diff
@@ -11,4 +11,5 @@ export enum DefaultValidationError {
   Number = 'number',
   Cron = 'cron',
   Ip2 = 'ip2',
+  InvalidPoolName = 'invalidPoolName',
```

Comment:
This is a very custom error and doesn't belong here.
It may be better to use `IxValidatorsService.customValidator`.